internal/gateway/gateway_test.go

```diff
@@ -61,19 +61,19 @@ func TestGateway(t *testing.T) {
 
 	gateways, err := InitGateways("test_targets.json", testTokenFunc, testJWTSecret)
 	assert.NoError(t, err)
-	assert.Equal(t, 2, len(gateways))
-	//make API request
+	assert.Equal(t, 5, len(gateways))
```

Comment:
Should we split up the tests now into separate methods? the test is a lot longer with the additions

Reply:
+1